android/res/layout/fragment_transfers.xml

```diff
@@ -18,12 +18,32 @@
  */
 -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:frostwire="http://schemas.android.com/apk/res-auto"
-              android:layout_width="match_parent"
-              android:layout_height="fill_parent"
-              android:orientation="vertical">
+    xmlns:frostwire="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
 
-    <com.frostwire.android.gui.views.ClearableEditTextView
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/basic_blue"
+        android:orientation="vertical">
+
+        <View
+            style="@style/divider_line_dark"
+            android:layout_width="match_parent"
+            android:layout_height="1dp" />
+
+        <!-- Dummy item to prevent EditText from receiving focus at startup -->
+        <LinearLayout
```

Comment:
what a trick, I would like to no need this (and the one in the search field too).